frontend/libs/studio-components-legacy/src/components/StudioExpression/SimplifiedEditor/LogicalExpressionEditor/SubExpression/SubExpressionValueSelector/SubExpressionValueContentInput/ComponentIdSelector.tsx

```diff
@@ -31,7 +31,7 @@ export const ComponentIdSelector = ({
 
   return (
     <Combobox
-      error={texts.errorMessages[errorKey]}
+      error={errorKey === null ? undefined : texts.errorMessages[errorKey]}
```

Comment:
Nitpick - I think using the `&&` operator is a more common way to conditionally pass props?


```suggestion
      error={errorKey && texts.errorMessages[errorKey]}
```

Reply:
Ah yes, you are correct.